src/sentry/integrations/aws_lambda/integration.py

```diff
@@ -316,9 +315,11 @@ def render_response(error=None):
 class AwsLambdaListFunctionsPipelineView(PipelineView):
     def dispatch(self, request: Request, pipeline) -> Response:
         if request.method == "POST":
-            # accept form data or json data
-            # form data is needed for tests
-            data = request.POST or json.loads(request.body)
+            raw_data = request.POST
+            data = {}
+            for key, val in raw_data.items():
+                # convert string to boolean
+                data[key] = val == "true"
```

Comment:
Is the expected value *always* a boolean? If so, I would change the comment to say that instead of what it's doing.